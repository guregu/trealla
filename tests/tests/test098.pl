:- initialization(main).
:- use_module(library(dif)).

main :-
	dif([],A), A=_*A,
	write_term(A,[max_depth(5)]), nl.

