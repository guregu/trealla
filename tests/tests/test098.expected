_4*(_4*(_4*(_4*(_4* ...))))
